core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/locks/DatastoreLockIndexEntry.java

```diff
@@ -143,6 +144,20 @@ public Builder timestamp(final Long timestamp) {
 			this.timestamp = timestamp;
 			return this;
 		}
+	
+		@JsonSetter
+		Builder description(final String description) {
+			// lock documents with description value should propagate their values to contexts
+			addContext(description);
+			return this;
+		}
+		
+		@JsonSetter
+		Builder parentDescription(final String parentDescription) {
+			// lock documents with a parentDescription value should propagate their values to contexts
+			addContext(parentDescription);
```

Comment:
`parentDescription` goes first in the list, so a `contexts.add(0, parentDescription)` equivalent could be used here. Although whether this is required or not will also depend on the order of the two setter calls.